### ISSUE TITLE
[a11y] Adding a Col Obj Attribute subview fails

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1624]
![firefox_Zq3ZawslPL](https://user-images.githubusercontent.com/37256050/168859787-7d5f41a3-e082-43b9-90a2-b5dbc90554b4.gif)

Adding a Col Obj Attribute subview on the default `kufish` CO form (in and out of a record set) causes it to load indefinitely.

The first time you add a CoAttr, it works. The second time you try, it loads forever.

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/168875216-967dd90b-b8a7-4ba4-972f-3be7cfb3c9b9.png)

Can be recreated across databases. `rgbe_herb`

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/168875400-803c4243-178a-4dfa-909b-ca57ca604a73.png)

Could apply to most subviews
